### PR TITLE
analog: add SetParameter command

### PR DESCRIPTION
**Pull request by DeadByElpy**
implement all parameters for GetParameter command

### CODE REVIEW

File: src/analog/utils/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -1277,15 +1277,29 @@ const deviceParameterConvertersMap = {
...
-        channel: getChannelValue(buffer)
-    }
+    {channel: getChannelValue(buffer)}
 );
 
 const setRequestChannelParameter = ( buffer: ICommandBinaryBuffer, parameter: IRequestChannelParameter ) => {
     setChannelValue(buffer, parameter.channel);
 };
 
+const getRequestDataTypeParameter = ( buffer: ICommandBinaryBuffer ): IRequestDataTypeParameter => (
```
Comment (DarkPark) [inline src/analog/utils/CommandBinaryBuffer.ts:1287]: do we need these 4 functions? they are used once each

File: src/analog/commands/uplink/setParameter.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+
+interface ISetParameterResponseParameters {
+    id: number;
+    status: number;
+}
+
+
+export const id: number = 0x03;
+export const name: string = 'setParameterResponse';
```
Comment (DarkPark) [inline src/analog/commands/uplink/setParameter.ts:39]:
```suggestion
export const name: string = 'setParameter';
```

File: src/analog/commands/downlink/setParameter.ts [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+ * const parameters = {
+ *     id: constants.deviceParameters.ABSOLUTE_DATA,
+ *     data: { value: 2023, meterValue: 204, pulseCoefficient: 100 }
+ * };
+ * const bytes = setParameter.toBytes(parameters);
+ *
+ * // command binary representation
+ * console.log(bytes);
+ * // output:
+ * // 03 0a 17 00 00 00 cc 83 00 00 07 e7
```
Comment (DarkPark) [inline src/analog/commands/downlink/setParameter.ts:21]:
```suggestion
 * 03 0a 17 00 00 00 cc 83 00 00 07 e7
```

Comment (DarkPark) [reply src/analog/commands/downlink/setParameter.ts:21]: thugh actually it's an array of numbers

File: src/analog/utils/CommandBinaryBuffer.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/analog/utils/CommandBinaryBuffer.ts]: functions removed

File: src/analog/commands/downlink/setParameter.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/analog/commands/downlink/setParameter.ts]: fixed

File: src/analog/commands/uplink/setParameter.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/analog/commands/uplink/setParameter.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**